src/com/projectkorra/projectkorra/waterbending/WaterBubble.java

```diff
@@ -15,15 +15,20 @@
 
 import com.projectkorra.projectkorra.ability.CoreAbility;
 import com.projectkorra.projectkorra.ability.WaterAbility;
+import com.projectkorra.projectkorra.attribute.Attribute;
 import com.projectkorra.projectkorra.configuration.ConfigManager;
 import com.projectkorra.projectkorra.util.TempBlock;
 
 public class WaterBubble extends WaterAbility {
 
-	private static long clickDuration; // How long the click variant lasts.
-	private static double maxRadius;
-	private static double speed;
-	private static boolean requireAir = false;
+	@Attribute("Click" + Attribute.DURATION)
+	private long clickDuration; // How long the click variant lasts.
```

Comment:
Is this comment necessary? The name is pretty self-documenting. Perhaps a javadoc would be better than a single line comment?